pages/ksphere/kommander/latest/clusters/attach-cluster/index.md

```diff
@@ -25,6 +25,13 @@ KEY_SIZE=4096
 PRIV_KEY_FILE=root-ca-private-key.pem
 CA_CERT_FILE=root-ca-certificate.pem
 
+case "$(uname -s)" in
+  Linux*)  base64Options="-w0";;
+  Darwin*) base64Options="-b 0";;
```

Comment:
AFAIK you don't need any options for Darwin: https://stackoverflow.com/a/46464081. I guess this issue has already been seen in the docs and we fixed it with `-b 0` thus we know it works too but can you please confirm?